build/docker/entrypoint.sh

```diff
@@ -10,6 +10,8 @@ initialize()
       return
     fi
 
+    export METRO_CONSUMEPLANE_ORDINALID=${HOSTNAME##*-}
```

Comment:
Did not understand the `##*-`.
Please explain.

Reply:
https://github.com/kubernetes/kubernetes/issues/40651
We need to fetch the ordinal ID for each pod of the stateful set.
Currently, k8s downstream APIs do not expose this field.
Hence the workaround - https://github.com/kubernetes/kubernetes/issues/40651#issuecomment-623593113